mne/viz/backends/_notebook.py

```diff
@@ -348,7 +363,7 @@ def show(self):
         except RuntimeError:
             # pyvista>=0.30.0
             viewer = self.plotter.show(
-                jupyter_backend="ipyvtk_simple", return_viewer=True)
+                jupyter_backend="ipyvtklink", return_viewer=True)
```

Comment:
Should this be a `try/except` as well?

Reply:
Since `0.30.0` pyvista does not `ipyvtk_simple` anymore

Reply:
Yes but I'm thinking that it should also work for people with `0.29` if possible

Reply:
I plan to bump/update those requirements in https://github.com/mne-tools/mne-python/pull/9341

Reply:
If it's only a line or two to keep it working, it would be nice not to force people to upgrade

Reply:
But if they have `ipyvtk_simple` and `pyvista < 0.30` it should work, right? For example:
```
kwargs = dict()
if LooseVersion(pyvista.__version__) < LooseVersion('0.30'):
    kwargs['jupyter_backend' ] = 'ipyvtk_simple'
else:
    kwargs['jupyter_backend'] = 'ipyvtklink'
viewer = self.plotter.show(return_viewer=True, **kwargs)
```

Reply:
> it would be nice not to force people to upgrade

I understand

Reply:
I'll rework this part to use `LooseVersion` instead.

Reply:
The API changed too, the following won't work:

```
self.plotter.show(jupyter_backend="ipyvtk_simple", return_viewer=True)
```